_dev/docker/ci-browsers/Dockerfile

```diff
@@ -7,7 +7,8 @@ ENV FXA_AUTO_INSTALL=0
 
 # Install playwright browsers
 # One of the biggest culprits of slow install times is playwright since it downloads browser binaries
-RUN yarn global add playwright; \
+RUN cd /home/circleci; \
+    npx playwright install; \
```

Comment:
We should use this command instead `npx playwright install --with-deps` It downloads deps required for CI.
[https://playwright.dev/docs/cli#install-system-dependencies)](https://playwright.dev/docs/cli#install-system-dependencies)

Reply:
I just tried adding `--with-deps` and it pulls in a lot of extra stuff. The base image for this is circleci's cimg/node:16.13-browsers, which comes with a bunch of libs preinstalled, and these are all cached in circleci's docker layer cache. As are result there's no overhead in using this image. 

By the way, this was working previously. _I think_ that a yarn upgrade may have caused the `yarn global add playwright` to stop working properly.